xml/FrameworksIndex/netcore-1.1.xml

```diff
@@ -29147,18 +29172,36 @@
       <Member Id="P:System.Threading.Tasks.UnobservedTaskExceptionEventArgs.Exception" />
       <Member Id="P:System.Threading.Tasks.UnobservedTaskExceptionEventArgs.Observed" />
     </Type>
+    <Type Name="System.Threading.Tasks.ValueTask" Id="T:System.Threading.Tasks.ValueTask">
+      <Member Id="M:System.Threading.Tasks.ValueTask.#ctor(System.Threading.Tasks.Sources.IValueTaskSource,System.Int16)" />
+      <Member Id="M:System.Threading.Tasks.ValueTask.#ctor(System.Threading.Tasks.Task)" />
+      <Member Id="M:System.Threading.Tasks.ValueTask.AsTask" />
+      <Member Id="M:System.Threading.Tasks.ValueTask.ConfigureAwait(System.Boolean)" />
+      <Member Id="M:System.Threading.Tasks.ValueTask.Equals(System.Object)" />
+      <Member Id="M:System.Threading.Tasks.ValueTask.Equals(System.Threading.Tasks.ValueTask)" />
+      <Member Id="M:System.Threading.Tasks.ValueTask.GetAwaiter" />
+      <Member Id="M:System.Threading.Tasks.ValueTask.GetHashCode" />
+      <Member Id="M:System.Threading.Tasks.ValueTask.op_Equality(System.Threading.Tasks.ValueTask,System.Threading.Tasks.ValueTask)" />
+      <Member Id="M:System.Threading.Tasks.ValueTask.op_Inequality(System.Threading.Tasks.ValueTask,System.Threading.Tasks.ValueTask)" />
+      <Member Id="M:System.Threading.Tasks.ValueTask.Preserve" />
+      <Member Id="P:System.Threading.Tasks.ValueTask.IsCanceled" />
+      <Member Id="P:System.Threading.Tasks.ValueTask.IsCompleted" />
+      <Member Id="P:System.Threading.Tasks.ValueTask.IsCompletedSuccessfully" />
+      <Member Id="P:System.Threading.Tasks.ValueTask.IsFaulted" />
+    </Type>
     <Type Name="System.Threading.Tasks.ValueTask`1" Id="T:System.Threading.Tasks.ValueTask`1">
       <Member Id="M:System.Threading.Tasks.ValueTask`1.#ctor(`0)" />
+      <Member Id="M:System.Threading.Tasks.ValueTask`1.#ctor(System.Threading.Tasks.Sources.IValueTaskSource{`0},System.Int16)" />
       <Member Id="M:System.Threading.Tasks.ValueTask`1.#ctor(System.Threading.Tasks.Task{`0})" />
       <Member Id="M:System.Threading.Tasks.ValueTask`1.AsTask" />
       <Member Id="M:System.Threading.Tasks.ValueTask`1.ConfigureAwait(System.Boolean)" />
-      <Member Id="M:System.Threading.Tasks.ValueTask`1.CreateAsyncMethodBuilder" />
```

Comment:
This is a little suspicious (method is being removed from .NET Core 1.1 and .NET Core 2.0 in the following file).